Dockerfile.prow

```diff
@@ -5,7 +5,7 @@ FROM registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux AS b
 WORKDIR /workspace
 COPY . .
 
-RUN if yum install -y prometheus-promu; then export BUILD_PROMU=false; fi && make common-build
```

Comment:
I think we should keep using the current one. say - https://github.com/openshift/prometheus/blob/c5b7496cfca7c5a2a4c6689093cdafaa36e83531/Dockerfile.ocp#L4
in DS, the external dependency should be included. for example: https://code.engineering.redhat.com/gerrit/gitweb?p=prometheus-alertmanager.git;a=blob;f=distgit/containers/prometheus-alertmanager/container.yaml.in;h=238085bd39163c9c7666b33792bb40f7feea9fca;hb=9cd0eea03c1ab3eaf777f4012dd816180ed29ab8